core/schema/src/main/liquibase/29.0.0/changelog.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+
+  <changeSet author="pschweizer" id="29.0.0-drop-timeseries_meta-table">
```

Comment:
We should include a note about breaking changes here: `docs/modules/releasenotes/pages/whatsnew.adoc`

Something like: Updated time series integration layer - need to use plugins compatible with OIA 0.6.x - DB tables are no longer used - no migration support.

Reply:
How do you like this? https://github.com/OpenNMS/opennms/pull/3520/commits/20add353adc4ef49e83bdcefa85a40533dcf9e4b